scripts/ci-test.sh

```diff
@@ -20,6 +20,24 @@ dockerBuild() {
   docker build -t interledger/five-bells-ledger .
 }
 
+mssqltest() {
+  : ${MSSQL_USERNAME?"MSSQL_USERNAME not set; note that CircleCI does not export environment variables on forks for security of secrets"}
+  MSSQL_DATABASE="CIRCLECI_$(openssl rand -hex 6)"
```

Comment:
Is the idea to have a different db each time? Is there any issue with using the same name each time?